src/Microsoft.AspNet.Mvc.Core/DefaultControllerModelConvention.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNet.Mvc.ApplicationModels;
+
+namespace Microsoft.AspNet.Mvc
+{
+    /// <summary>
+    /// The class used by <see cref="ApplicationModelConventionExtensions"/> to add a
+    /// <see cref="IControllerModelConvention"/> to all the controllers in the application.
+    /// </summary>
+    public class DefaultControllerModelConvention : IApplicationModelConvention
```

Comment:
I don't really love the name here, but I don't have a better idea. Maybe `ControllerApplicationModelConvention`? It's pretty bad too.